code/src/styled-components.js

```diff
@@ -0,0 +1,144 @@
+import styled from 'styled-components/macro';
+
+export const Container = styled.div`
+    width: clamp(250px, 90%, 350px);
+    margin: 3rem auto 4rem auto;
+
+    @media (min-width: 668px) {
+        width: clamp(250px, 90%, 500px);
+    }
+`
+
+export const Header = styled.header`
+    margin: 0;
+    margin-bottom: 1rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    h1 {
+        color: hsl(21,84%,54%);
+        font-size: 3rem;
+        margin: 0;
+    }
+` 
+
+export const Form = styled.form`
+    font-size 2rem;
+    color: hsl(300, 30%, 60%);
+    text-align: center;
+    width: 100%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-content: center;
+    gap: 25px;
+
+    input {
+        padding: .5em;
+        font-size: 1rem;
+        flex-grow: 4;
+        border: 1px solid hsl(21,84%,54%);
+        border-radius: .25rem;
+    }
+`
+
+export const NewTaskButton = styled.button`
+    font-weight: 700;
+    font-family: 'Nunito';
+    text-transform: uppercase;
+    color: white;
+    min-width: 10ch;
+    background-color: hsl(21,84%,54%);
+    border-radius: .25rem;
+    border: none;
+    cursor: pointer;
+`
+
+export const TaskCounter = styled.div`
+    display: grid;
+    font-size: clamp(1rem, 5vw + 1rem, 1.5rem);
+`
+export const TaskItem = styled.div`
+    margin: 0 auto;
+    width: 100%;
+`
+
+export const SmallButton = styled.button`
+    font-family: 'Nunito';
+    position: absolute;
+    top: 1rem;

```

Comment:
When adding a long to do - the button hides a little bit of the content. Maybe it could be solved if placing it with a lower margin from top.